src/exclusive-or.rst

```diff
@@ -148,10 +146,10 @@ XOR of the two sequences of bits.
    :align: center
    :alt: OTP
 
-If an attacker sees the ciphertext, we can prove that they will learn
-zero information about the plaintext without the key. This property is
-called *perfect security*. The proof can be understood intuitively by
-thinking of XOR as a programmable inverter, and then looking at a
+If an attacker sees the ciphertext, we can prove that
+zero information is learned about the plaintext without the key. This property is
+called *perfect security*. The proof can be understood intuitively. 
```

Comment:
Whoa! I get that this doesn't change that word but this should absolutely say "perfect secrecy" not "perfect security".

Reply:
Oh right. It's a cryptography term